infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -545,7 +545,7 @@ export class AtatWebApiStack extends cdk.Stack {
       database: this.database,
       ormLayer: this.ormLayer,
       functionPropsOverride: {
-        timeout: Duration.seconds(10),
+        timeout: cdk.Duration.seconds(10),
```

Comment:
This makes it similar to other uses of same

Reply:
I thought at one point we were going the other way, but fair enough the others are also using `cdk.XX` too, so better to stay consistent.